tests/countries/test_guam.py

```
@@ -55,3 +47,62 @@
+            ("2022-09-05", "Labor Day"),
+            ("2022-10-10", "Columbus Day"),
+            ("2022-10-31", "Halloween"),
+            ("2022-11-02", "All Souls' Day"),
+            ("2022-11-11", "Veterans Day"),
+            ("2022-11-24", "Thanksgiving Day"),
+            ("2022-12-08", "Lady of Camarin Day"),
+            ("2022-12-25", "Christmas Day"),
+            ("2022-12-26", "Christmas Day (observed)"),
+        )
+
+    def test_l10n_th(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)